StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -354,6 +355,24 @@ extension LinkAccountPickerViewController: LinkAccountPickerBodyViewDelegate {
 
         let selectedPartnerAccount = selectedAccountTuple.partnerAccount
 
+        if let drawerOnSelection = selectedAccountTuple.accountPickerAccount.drawerOnSelection {
+            let genericInfoViewController = GenericInfoViewController(
+                genericInfoScreen: drawerOnSelection,
+                panePresentationStyle: .sheet,
+                didSelectPrimaryButton: { [weak self] in
+                    self?.lastDrawerOnSelectionViewController?.dismiss(animated: true)
```

Comment:
[Code logic for what happens on dismiss](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/pull/21998/files#diff-7a6a4e0274945303bb90b85dd96ba66a5a9fc9cb57273504e552729b20d27054R23-R31)

(I link code so reviewer can help double check)